build.js

```diff
@@ -192,12 +192,21 @@ function generateTableRows(writer, maximumLevel, parentKeys, node) {
             rowWriter.cell((cellContentWriter) => {
               let empty = true;
               if (documentationUrls) {
+                if (synopsis) {
+                  // With Synopsis and Documentation URLs we need a more complex layout
+                  cellContentWriter.write('<div class="flex flex-row">');
+                  cellContentWriter.write(`<div class="grow">${marked.parse(synopsis)}</div>`);
+                  cellContentWriter.write('<div class="grow-0">');
+                }
                 cellContentWriter.write(documentationUrls
                   .map((element) => `<a class="btn btn-blue" href="${element}" target="_blank" rel="noopener">docs</a>`)
                   .join(' '));
+                if (synopsis) {
+                  cellContentWriter.write('</div></div>');
```

Comment:
This bit looks a little strange to me, if you're not appending any content to this div tag then there's probably a better way to do this. That being said, if it's not worth the effort then feel free to ignore :)

Reply:
Does this help?: https://github.com/ably/features/pull/48/commits/35682e96aed34b0ca9c95abf4e699ef15d08765c

Reply:
Ahhh, I didn't realise they were both closing tags so I thought you were adding an empty div to trigger a line break or something 😅

That commit does make it clearer, thanks for that 👍